src/components/webhooks/WebHookSettings.tsx

```diff
@@ -53,49 +56,115 @@ interface Props extends RouteComponentProps, WithStyles<typeof styles> {
 
 function WebHookSettings(props: Props) {
   let [expanded, setExpanded] = useState(false);
-  let [initialURL, setInitialURL] = useState(props.info.webhookSettings.webhookURL || '');
-  let [inputValue, setInputValue] = useState(props.info.webhookSettings.webhookURL || '');
+  let [initialWebhookURL, setInitialWebhookURL] = useState(props.info.webhookSettings.webhookURL || '');
```

Comment:
If you'll add organization id to the response of `securedVariableMutation` then Relay will automatically update properties from the response and you don't need to use a separate state for the initial value.

Reply:
> If you'll add organization id to the response of `securedVariableMutation`

You mean the client mutation ID?

Reply:
The one in `securedVariableMutation` varialble.

```GraphQL
info {
  id // here
  webhookSettings {
    webhookURL
    maskedSecretToken
  }
}
```

Reply:
See https://github.com/cirruslabs/cirrus-ci-web/pull/390/commits/2083ac49889e2c19948845089df60fe537b26ed1.

Looks like we don't even need to add the `id`, it gets updated anyway.